compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/SkiaParagraph.skiko.kt

```diff
@@ -188,15 +188,15 @@ internal class SkiaParagraph(
     private fun lineMetricsForOffset(offset: Int): LineMetrics? {
         checkOffsetIsValid(offset)
         val metrics = lineMetrics
-        for (line in metrics) {
-            if (offset < line.endIncludingNewline) {
-                return line
-            }
-        }
-        if (metrics.isEmpty()) {
+        if (metrics.isEmpty())
```

Comment:
Please keep curly braces even for a single statement ([we already discussed this before](https://github.com/JetBrains/compose-multiplatform-core/pull/555#discussion_r1191159275)).